packages/energy-api-influxdb/test/app.e2e-spec.ts

```diff
@@ -154,4 +154,28 @@ describe('ReadsController (e2e)', () => {
         expect(reads[0].value).to.equal((2500 - 1500) * 10 ** 3);
       });
   });
+
+  it('should return last meter read', async () => {
+    const today = new Date();
+    today.setHours(today.getHours() - 2);
+    const date = today.toISOString();
+    const measurement = new MeasurementDTO();
+    measurement.unit = Unit.kWh;
+    measurement.reads = [{ timestamp: today, value: 1750 }];
+
+    await request(app)
+      .post('/meter-reads/M1')
+      .send(measurement)
+      .expect(HttpStatus.CREATED);
+
+    await request(app)
+      .get('/meter-reads/M1/latest')
+      .expect(200)
+      .expect((res) => {
+        const reads = res.body as ReadDTO[];
```

Comment:
Since latest is expected to return single read there is no need to return [].

Reply:
Yeah but in the service it is reusing the execute method which returns ReadDTO[].

Reply:
I guess then we shouldn't forward the whole array to the upper levels, just extract one ReadDTO from the .execute call and forward that one.

Reply:
Yeah of course. I wanted the new method to be really similar with the others.
It should be ok now